<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomaticCaptureDialog</class>
 <widget class="QDialog" name="AutomaticCaptureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Automatic Capture</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Automatic capture options:</string>
     </property>
     <widget class="QRadioButton" name="partialDiscRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Partial disc capture</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="leadInCaptureRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Lead-in to address capture</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="wholeDiscRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Whole disc capture</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="captureStatusPreLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="captureStatusLabel">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>100</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cavGroupBox">
     <property name="title">
      <string>CAV capture:</string>
     </property>
     <widget class="QLabel" name="startFrameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start frame:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="endFrameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>End frame:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="startCavCapturePushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Start CAV capture</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="startFrameLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="endFrameLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="clvGroupBox">
     <property name="title">
      <string>CLV capture:</string>
     </property>
     <widget class="QLabel" name="endTimeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>End time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="startTimeLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Start time:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="startClvCapturePushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Start CLV capture</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTimeEdit" name="startTimeTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>118</width>
        <height>25</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="endTimeTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>118</width>
        <height>25</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>wholeDiscRadioButton</tabstop>
  <tabstop>partialDiscRadioButton</tabstop>
  <tabstop>leadInCaptureRadioButton</tabstop>
  <tabstop>startFrameLineEdit</tabstop>
  <tabstop>endFrameLineEdit</tabstop>
  <tabstop>startCavCapturePushButton</tabstop>
  <tabstop>startTimeTimeEdit</tabstop>
  <tabstop>endTimeTimeEdit</tabstop>
  <tabstop>startClvCapturePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
